### DISCUSSION TITLE
Lesson 4 | int256 price issue

### DISCUSSION THREAD

**Discussion by MorCeketliSaldiray** [Cyfrin/foundry-full-course-cu#1349] [Q&A]
I'm on lesson 4. I use Chainlink's doc. The link is [here.](https://docs.chain.link/data-feeds/using-data-feeds#solidity) As you will see, it use `int answer` for `getChainlinkDataFeedLatestAnswer`. But when I use `(,int256 answer,,,) = dataFeed.latestRoundData();` it cause error. When I change it with `int256 price` as Patrick used, it works. Is documentation wrong or am I?

**Comment by developer-maxim** [comment]
Did you note that the variable getting returned is also *answer* in the code below? It might be the case that you returned *price* but used *answer* instead. This might be the error, glad to help with more details :)

```solidity
function getChainlinkDataFeedLatestAnswer() public view returns (int) {
        // prettier-ignore
        (
            /* uint80 roundID */,
            int answer,
            /*uint startedAt*/,
            /*uint timeStamp*/,
            /*uint80 answeredInRound*/
        ) = dataFeed.latestRoundData();
        return answer;
    }
    ```
